lib/order.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/menu'
+
+class Order
+
+  attr_reader :basket, :menu, :total
+
+  def initialize
+    @basket = []
+    @menu = Menu.new
+    @total = 0
```

Comment:
Good instance variables. Names are short but their functions are easily decipherable by looking at the name